### PR TITLE
(To be discussed) (Idea) feature: multi dimensional reward

### PR DESCRIPTION
**Pull request by YuriCat**
Do we delete OUTCOME, or use OUTCOME as the first dimension of REWARD if it is defined?

### CODE REVIEW

**Comment by ikki407** [comment]
My first impression was that this change could be closer to the implementation of general reinforcement learning. Also, it can be a simple code 👍 
However, I think a user must pay attention to the first dimension of multi-reward is the outcome. How can we set `gamma` 
 in games with no outcome? like `gamma: [1, 0.99]`?

**Comment by YuriCat** [comment]
Yes, `gamm: [1, 0.99]` will work.
Warnings for over length need to be added.

File: handyrl/train.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def compose_losses(outputs, log_selected_policies, total_advantages, targets, ba
     entropy = dist.Categorical(logits=outputs['policy']).entropy().mul(tmasks.sum(-1))
     losses['ent'] = entropy.sum()
 
-    base_loss = losses['p'] + losses.get('r', 0)
+    base_loss = losses['p'] + losses.get('v', 0)
```
Comment (ikki407) [inline handyrl/train.py:211]: wow, thank you
